virttest/utils_test/qemu/__init__.py

```diff
@@ -126,20 +126,20 @@ def setup_win_driver_verifier(session, driver, vm, timeout=300):
     :param vm: VM object.
     :param timeout: Timeout in seconds.
     """
-    verifier_status = _check_driver_verifier(session, driver)[0]
-    if not verifier_status:
-        error_context.context("Enable %s driver verifier" % driver,
-                              logging.info)
+    error_context.context("Enable %s driver verifier" % driver,
+                          logging.info)
+    verifier_flags = vm.params.get("verifier_flags")
+    if verifier_flags:
+        verifier_setup_cmd = "verifier /flags %s /driver %s.sys" % (
+                verifier_flags, driver)
+    else:
         verifier_setup_cmd = "verifier /standard /driver %s.sys" % driver
-        session.cmd(verifier_setup_cmd,
-                    timeout=timeout,
-                    ignore_all_errors=True)
-        session = vm.reboot(session)
-        verifier_status, output = _check_driver_verifier(session, driver)
-        if not verifier_status:
-            msg = "%s verifier is not enabled, details: %s" % (driver,
-                                                               output)
-            raise exceptions.TestFail(msg)
+    session.cmd(verifier_setup_cmd, timeout=timeout, ignore_all_errors=True)
+    session = vm.reboot(session)
```

Comment:
Can we skip rebooting when the settings have been done in the previous tests?

Reply:
Yes, then we need to check flags.
Now it's modified, thanks